Bottom: e20ba96849142cd0504e2bbd7aae5f157a179bf4
Top:    7b9931f8442f8dfcbebda65370df11206d54409c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:37:56 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 41babc6..da0de99 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1612,6 +1612,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
